<launch>
    <!-- Node for the robot controller -->
    <arg name="base_link_diameter" default="0.1" />
    <arg name="max_speed" default="1.5" />
    
    <node name="robot_controller" pkg="web_control" type="main.py" output="screen">
        <param name="base_link_diameter" value="$(arg base_link_diameter)" />
        <param name="max_speed" value="$(arg max_speed)" />
        <param name="use_sim_time" value="false"/>
    </node>
</launch>

